benchmarks/filter_2d.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+from pyinstrument import Profiler
+
+from cellfinder_core.detect.filters.plane import TileProcessor
+from cellfinder_core.detect.filters.setup_filters import setup_tile_filtering
+
+# Use random 16-bit integer data for signal plane
+shape = (10000, 10000)
+
+signal_array_plane = np.random.random(shape)
+signal_array_plane = (signal_array_plane * 65535).astype(np.uint16)
```

Comment:
```suggestion
signal_array_plane = np.random.randint(low=0, high=65535, size=shape, dtype=np.uint16)
```
Is a bit clearer (I discovered `randint` after writing the original benchmark)